statistics/update.go

```diff
@@ -156,15 +156,40 @@ func (h *Handle) NewSessionStatsCollector() *SessionStatsCollector {
 	return newCollector
 }
 
-// DumpStatsDeltaToKV sweeps the whole list and updates the global map. Then we dumps every table that held in map to KV.
-func (h *Handle) DumpStatsDeltaToKV() error {
+var (
+	// DumpStatsDeltaRatio is the lower bound of `Modify Count / Table Count` for stats delta to be dumped.
+	DumpStatsDeltaRatio = 1 / 10000.0
+	// dumpStatsMaxDuration is the max duration since last update.
+	dumpStatsMaxDuration = time.Hour
```

Comment:
should we make this two param into configuration file if it maybe need some tuning after program released?

Reply:
I think these parameters are not that important, put them into the configuration file would expose too much internal implementation. If it is needed in the future, we may put them in the config file at that time.